pantheon/src/main/java/org/web3j/protocol/pantheon/response/StructLogs.java

```diff
@@ -0,0 +1,94 @@
+package org.web3j.protocol.pantheon.response;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Map;
+
+public class StructLogs {
+
+    private int pc;
+    private String op;
+    private int gas;
+    private int gasCost;
+    private int depth;
+    private List<String> stack;
+    private List<String> memory;
+    private Map<BigInteger, String> storage; // maybe List<Map<BigInteger, String>>
```

Comment:
Any need to keep this comment here?

Reply:
There is definitely no need for that. I'll remove it in the next commit.